src/test/scala/com/eclipsesource/schema/FormatSpec.scala

```diff
@@ -63,28 +63,49 @@ class FormatSpec extends Specification with JsonSpec {
     }
 
     "validate int32 format" in {
-      val schema = JsonSource.schemaFromString(
+      val integerSchema = JsonSource.schemaFromString(
+        """
+          |{
+          |  "type": "integer",
+          |  "format": "int32"
+          | }
+        """.stripMargin).get
+      val numberSchema = JsonSource.schemaFromString(
         """
           |{
           |  "type": "number",
           |  "format": "int32"
           | }
         """.stripMargin).get
-      validator.validate(schema, JsNumber(Integer.MAX_VALUE)).isSuccess must beTrue
```

Comment:
Would it be good to check Int.MinValue too? Also, I thought that scala prefers Int.MaxValue, I understand Integer.MAX_VALUE comes from java..